includes/class-amp-theme-support.php

```diff
@@ -2104,7 +2104,7 @@ public static function prepare_response( $response, $args = [] ) {
 		$enable_response_caching = (
 			$enable_response_caching
 			&&
-			! AMP_Validation_Manager::should_validate_response()
+			! AMP_Validation_Manager::$is_validate_request
```

Comment:
I went with `validate` because query param is `amp_validate` and there are methods like: 
* `\AMP_Validation_Manager::get_amp_validate_nonce()`
* `\AMP_Validation_Manager::get_validate_url_error_message()`
* `\AMP_Validation_Manager::should_validate_response()`

The outlier here is `\AMP_Validation_Manager::get_validation_response_data()`. So to be consistent, I think actually it should be renamed to `\AMP_Validation_Manager::get_validate_response_data()`.

Reply:
You don't really care about grammar, do you...? 😜

Reply:
See 792f6bb223bab3f99a54bdd837a020b8f0a67d51. 

I don't really see it as being ungrammatical. It's a “request to validate” aka a “validate request”, like a “request to pull” is also a “pull request”.

This is somewhat of an attempt to distinguish validation which happens with generation of every AMP page, with the “deep” validation that obtains the source information for the underlying validation errors. In other words, `validation` happens all the time as a side effect of requesting an AMP page, but `validate` only happens if you are explicitly requesting the validation results.

Where I know you will agree is that `AMP_Validation_Manager` is doing too much and needs to be split up. So we'll be sure to revisit the terminology when we do that. 😄

Reply:
Ha, dangling refactoring carrots in front of me now...

Alright, works for me! 😄